docs/design/minsuccess.md

```diff
@@ -0,0 +1,14 @@
+# MinSuccess
+
+## Introduction
```

Comment:
Suggest to complete the design doc with more details officially. Please refer to https://github.com/volcano-sh/volcano/blob/master/docs/design/queue-resource-reservation-design.md as example.